src/Orleans/CodeGeneration/IGrainMethodInvoker.cs

```diff
@@ -11,6 +11,8 @@ public interface IGrainMethodInvoker
         /// <summary> The interface id that this invoker supports. </summary>
         int InterfaceId { get; }
 
+        ushort InterfaceVersion { get; }
```

Comment:
From my testing with our serializer, there is a real beneif in using ushort, it takes less space. I checked in `BuiltInTypes.SerializeInvokeMethodRequest` the byte array that resulted from the serialization.